azurelinuxagent/common/conf.py

```diff
@@ -150,7 +154,12 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "HttpProxy.Port": None,
     "ResourceDisk.SwapSizeMB": 0,
     "Autoupdate.Frequency": 3600,
-    "Logs.CollectPeriod": 3600
+    "Logs.CollectPeriod": 3600,
+    #
+    # "Debug" options are experimental and may be removed in later
+    # versions of the Agent.
+    #
+    "Debug.CgroupCheckPeriod": 300,
```

Comment:
Nice! Small thought, could you group all the `Debug` options together? Makes it easier to read and categorize.

Reply:
The current code groups them by type so I would need to create 3 parallel arrays (one for each type) and add code to traverse those arrays to several functions.

Hopefully that comment help finding the Debug options easier.